<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SvrCtrlPage</class>
 <widget class="QWidget" name="SvrCtrlPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_9">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>客体重用控制</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="cur_audstatus_label_2">
        <property name="text">
         <string>当前状态:      无接口，未实现</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="close_client_reuse_Button">
        <property name="text">
         <string>开启/关闭客体重用</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_10">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>完整性策略</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="cur_audstatus_label_3">
        <property name="text">
         <string>当前状态:      无接口，未实现</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="close_client_reuse_Button_2">
        <property name="text">
         <string>开启/关闭完整性验证</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>591</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_11">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>SAK功能开关</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_16">
      <item row="0" column="0">
       <widget class="QLabel" name="cur_sakstatus_label">
        <property name="text">
         <string>当前状态:      无接口，未实现</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="open_close_sak_Button">
        <property name="text">
         <string>开启/关闭当前SAK功能</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="def_sakstatus_label">
        <property name="text">
         <string>当前状态:      无接口，未实现</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="open_close_def_sak_Button">
        <property name="text">
         <string>开启/关闭默认SAK功能</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>57</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="openAllButton">
     <property name="text">
      <string>全部开启</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_7">
     <property name="title">
      <string>安全策略控制</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="cur_status_label">
        <property name="text">
         <string>当前状态:enforcing</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="open_closeSecPolButton">
        <property name="text">
         <string>开启/关闭安全策略</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
